src/Requirement/Requirement.js

```diff
@@ -15,48 +16,34 @@ class Requirement extends React.Component {
     }
 
     return requirement;
-  }
+  };
 
-  renderStyles() {
-    const styles = {
-      root: {
-        float: 'left',
+  get style() {
+    const style = {
+      base: {
         width: '100%',
         height: 'auto',
-      },
-      asterisk: {
-        float: 'left',
-        width: '10px',
-        height: '11px',
-        backgroundImage: `url(${require('./asterisk.svg')})`,
-        backgroundSize: '10px 11px',
-        backgroundRepeat: 'no-repeat',
+        fontFamily: 'Source Sans Pro',
+        fontSize: '12px',
+        color: '#A7AAB2',
       },
     };
 
-    return styles;
-  }
-
-  renderAsterisk(requirement, styles) {
-    if (requirement[0] === 'required') {
-      return (
-        <span style={styles.asterisk}></span>
-      );
-    }
-
-    return false;
-  }
+    return style;
+  };
 
   render() {
-    const styles = this.renderStyles();
-    const requirement = this.getRequirement();
+    const isRequired = this.requirement[0] === 'required';
 
     return (
-      <div style={styles.root}>
-        {this.renderAsterisk(requirement, styles)}
+      <div style={this.style.base}>
```

Comment:
Is it possible (or wanted) to return nothing here instead of an empty `div` when an item is not required?